extensions/wikia/Wall/WallThread.class.php

```diff
@@ -215,6 +217,7 @@ public function getLastMessage() {
 			);
 			return $row;
 		} );
+		$test = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$test" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1481)